<launch>

  <arg name="uav_name" default="$(optenv UAV_NAME uav)"/>

  <group ns="$(arg uav_name)">
    
    <node pkg="cnn_detect" name="uav_detection" type="filter_single" output="screen" >
      
      <rosparam file="$(find cnn_detect)/config/filter_single.yaml" />
      <param name="uav_name" value="$(arg uav_name)" />
      <param name="calib_path" value="$(find cnn_detect)/config/mobius_calib.txt" />
    
      <remap from="~detections" to="uav_detect/detections" />

    </node>
  
  </group>

</launch>
